tests/decodeoutput.cpp

```diff
@@ -59,6 +59,9 @@ Decode_Status DecodeOutput::processOneFrame(bool drain)
 {
     Decode_Status status = renderOneFrame(drain);
 
+    if (status == RENDER_NO_AVAILABLE_FRAME)
```

Comment:
we only renderDone picture when status == RENDER_SUCCESS. Can we change code to this
...
if (status == RENDER_SUCCESS) {
      m_renderFrames++;
      m_decoder->renderDone....
}
return status;
}